schemas/adaptive-card.json

```diff
@@ -866,6 +1110,26 @@
 				"id"
 			]
 		},
+		"InlineAction": {
+			"additionalProperties": true,
+			"type": "object",
+			"version": "1.2",
+			"description": "The inline action for the input. Typically displayed to the right of the input. It is strongly recommended that an icon on the action (which will be displayed instead of the title of the action).",
+			"oneOf": [
+				{
+					"$ref": "#/definitions/Action.Submit"
+				},
+				{
+					"$ref": "#/definitions/Action.ShowCard"
+				},
```

Comment:
As far as I'm aware, inlineAction does NOT support ShowCard (although it would make sense if it did). I checked JS, UWP, and WPF and they all don't support ShowCard for inlineAction.